contrib/opentelemetry/handler.go

```diff
@@ -130,12 +130,20 @@ func (m MetricsHandler) Counter(name string) client.MetricsCounter {
 func (m MetricsHandler) Gauge(name string) client.MetricsGauge {
 	// TODO(https://github.com/open-telemetry/opentelemetry-go/issues/3984) replace with sync gauge once supported
 	var config atomic.Value
-	config.Store(0.0)
-	_, err := m.meter.Float64ObservableGauge(name,
-		metric.WithFloat64Callback(func(ctx context.Context, o metric.Float64Observer) error {
```

Comment:
Hrmm, were you able to dig up what changed on their side for why this callback doesn't work for us anymore? I am just curious, if not no worries.

Reply:
I wasn't able to find what exact commit broke it unfortunately